zerver/tests/test_users.py

```diff
@@ -1979,6 +1980,36 @@ def test_stream_recipient_info(self) -> None:
         self.assertEqual(info.default_bot_user_ids, {normal_bot.id})
         self.assertEqual(info.all_bot_user_ids, {normal_bot.id, service_bot.id})
 
+        # Now Hamlet follows the topic with 'followed_topic_email_notifications' global setting enabled.
+        hamlet.enable_followed_topic_email_notifications = True
+        hamlet.save()
```

Comment:
We should instead use `do_change_user_setting` here instead of manually changing the DB.
(Not high priority, though also a ping that it would be worthwhile to split this test suite whenever you are free :))

Reply:
Hm, used `do_change_user_setting`.